instrumentation/rspec/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTelemetry RSpec Instrumentation
+
+The RSpec instrumentation is a community-maintained instrumentation for the [RSpec][rspec-home] BDD testing framework.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-instrumentation-rspec
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-instrumentation-rspec` in your `Gemfile`.
+
+## Usage
+
+To use the instrumentation, call `use` with the name of the instrumentation:
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::Rspec'
+end
+```
+
+Alternatively, you can add it directly to RSpec as a customer formatter:
+
+```ruby
+RSpec.configure do |config|
+  config.formatter = OpenTelemetry::Instrumentation::RSpec::Formatter
+end
+```
+
+### Using multiple tracer providers
```

Comment:
This is great, i appreciate the detail here. My only nit is I think it should be encouraged to `Always` sample, since a failing test that gets dropped due to random sampling sort've defeats the whole point of rspec instrumentation imo. I guess that's the implicit default behavior in the code example anyway, but I wouldn't mind seeing this be made explicit.

Reply:
I’ve added something for this